.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+
+updates:
+  - package-ecosystem: "github-actions"
```

We want to look for updates to go deps for this repo, don't we?
```suggestion
  - package-ecosystem: "gomod"
```